packages/common/src/types/field.ts

```diff
@@ -264,7 +264,7 @@ export const friendlyName = (text: string): string => {
         return text;
     }
     return [
-        capitalize(first.toLowerCase()),
+        capitalize(first!.toLowerCase()),
```

Comment:
Did you get any typescript errors? Didn't get any when I removed this.

Reply:
Yes, my linter marks this as an error because it considers the variable as possibly undefined. So I'm using this "hard check"

Reply:
We added a check above this line that makes sure `first` is not undefined/null.
Removed this change from the PR.